lib/desafioElixirAPI_web/controllers/operation_controller.ex

```diff
@@ -0,0 +1,54 @@
+defmodule DesafioElixirAPIWeb.OperationController do
+  @moduledoc """
+  Handles all Operation-related HTTP requests.
+  """
+
+  use DesafioElixirAPIWeb, :controller
+
+  alias DesafioElixirAPI.{Operation, User}
+  alias DesafioElixirAPIWeb.FallbackController
+
+  action_fallback FallbackController
+
+  def create(conn, params) do
+    with {:ok, multi} <- DesafioElixirAPI.create_operation(params) do
+      conn
+      |> put_status(:ok)
```

Comment:
```suggestion
      |> put_status(:created)
```
[Lista de atoms possíveis](https://hexdocs.pm/plug/Plug.Conn.Status.html#code/1)